Library/docs/source/learn/users/management.md

```diff
@@ -0,0 +1,302 @@
+
+* `ForgotPassword`: This method initiates the password reset flow by allowing the user to request a reset link. CSP expects the services to send an email with instructions on how to reset the password.
+
+* `ResetUserPassword`: This method is then used by users after verification (performed by them receiving and interacting with a reset link via email).
+
+
+
+### Step-by-Step Guide to Resetting a Password
+
+**Step 1: Confirming the User's Email**
+
+Before resetting the password, confirm the user's email address. This step ensures that the reset request comes from the legitimate user.
+
+**Step 2: Invoking ForgotPassword**
+
+Use the `ForgotPassword` method to initiate the password reset process. The system will inform the services and expect a reset email to be sent to the user containing a link and token that allows them to change their password.
+
+The method also accepts three other parameters that govern where the user is taken during the password reset flow.
+
+* `EmailLinkUrl`: Optional email link url including the environment to land on that URL directly.
+* `RedirectUrl`: Optional redirect URL to embed in the generated email link.
+* `UseTokenChangePasswordUrl`: Whether to have the link in the email direct to the Token Change URL.
+
+```
+void ForgotPassword(const csp::common::String& Email,
+				const csp::common::Optional<csp::common::String>& RedirectUrl,
+				const csp::common::Optional<csp::common::String>& EmailLinkUrl,
+				bool UseTokenChangePasswordUrl,
+				NullResultCallback Callback);
+```
+
+**Step 3: Implementing a Password Reset Form or Interface**
+
+Once the user receives the reset email, they click the link to a password reset form. The form typically prompts the user to enter and confirm a new password. Implement a simple and user-friendly interface to ensure users can easily update their passwords without confusion.
+
+**Step 4: Completing the Password Reset**
+
+After the user submits the form with a new password, the application can submit the verification token retrieved via the email, the user's ID, and the new password to CSP by invoking the `ResetUserPassword` method.
+
+```
+void ResetUserPassword(const csp::common::String& Token,
+			    const csp::common::String& UserId,
+			    const csp::common::String& NewPassword,
+			    NullResultCallback Callback);
+```
+
+The application should pass in the token obtained via the email expected to be sent by the services (after invoking `ForgotPassword`).
+
+## Updating Display Name
+
+Keeping user information accurate and updated is important for a good user experience within CSP. Allowing users to update their display name provides flexibility and personalization. This feature helps users maintain current and relevant profiles, reflecting changes in their roles, preferences, or identities. A personalized display name ensures that users are easily recognizable within the platform, enhancing communication and collaboration.
```

Comment:
[consider] "This feature helps users maintain current and relevant profiles, reflecting changes in their roles, preferences, or identities." implies that users have several profiles, roles and identities each. Consider removing the plural, so "This feature helps users maintain a current and relevant profile, reflecting changes in their role, preferences, or identity."